client/src/rpc_cache.rs

```diff
@@ -0,0 +1,74 @@
+use crate::rpc_config::RpcLargestAccountsFilter;
+use crate::rpc_response::RpcAccountBalance;
+use std::collections::HashMap;
+use std::time::{Duration, SystemTime};
+
+#[derive(Debug, Clone)]
+pub struct LargestAccountsCache {
+    duration: u64,
+    cache: HashMap<Option<RpcLargestAccountsFilter>, LargestAccountsCacheValue>,
+}
+
+#[derive(Debug, Clone)]
+struct LargestAccountsCacheValue {
```

Comment:
Might not be relevant, but if the explorer is set up to update largest accounts in real time, this could prevent the data going backward in time if it hits a different api endpoint with an older cache

Reply:
Good idea, just added.